### PR TITLE
misc fixes

### PR DESCRIPTION
**Pull request by vicb**

### CODE REVIEW

File: modules/@angular/compiler/test/html_parser_spec.ts [hunk 1]
```diff
@@ -120,6 +121,18 @@ export function main() {
               ]);
         });
 
+        it('should special case ng-container when adding a required parent', () => {
```
Comment (tbosch) [inline modules/@angular/compiler/test/html_parser_spec.ts]: Could you add a test for the case where the `<thead>` is missing, i.e. a `<tbody>` should be inserted? The `<tbody>` should be probably inserted above the `<ng-container>` then...

File: modules/@angular/compiler/src/schema/dom_element_schema_registry.ts [hunk 2]
```diff
@@ -265,7 +265,15 @@ export class DomElementSchemaRegistry extends ElementSchemaRegistry {
 
   hasProperty(tagName: string, propName: string): boolean {
     if (tagName.indexOf('-') !== -1) {
-      // can't tell now as we don't know which properties a custom element will get
+      if (tagName == 'ng-container') {
+        return false;
+      }
+
+      if (tagName == 'ng-content') {
```
Comment (tbosch) [inline modules/@angular/compiler/src/schema/dom_element_schema_registry.ts]: For `ng-select` we should never allow a property binding. E.g.
`<ng-content [select]="...">` is wrong as we evaluate the `select` during compilation.

I.e. this should be:
```
if (tagName == 'ng-content') {
  return false;
}
```

File: modules/@angular/compiler/test/html_parser_spec.ts (hunk 1 above)
Comment (vicb) [inline modules/@angular/compiler/test/html_parser_spec.ts]: good catch, thanks !

**Comment by vicb** [comment]
merged via 1eaa193 and previous commits

**Pull request closed without merging**